packages/extension/src/ui/features/actions/ApproveTransactionScreen.tsx

```diff
@@ -280,7 +280,8 @@ export const ApproveTransactionScreen: FC<ApproveTransactionScreenProps> = ({
   }, [selectedAccount, buildResult, onSubmit, onReject, navigate])
 
   if (!selectedAccount) {
-    throw Error(`No account found for network ${networkId}`)
+    rejectAction(actionHash)
+    return <Navigate to={routes.accounts()} />
```

Comment:
Why clicking the reset button would trigger this component?

Reply:
I can't reproduce at all with the reset button, even resetting when we are on the approval page.
Does it end up in this component every time you click the reset button?

Reply:
This is what clicking reset button shows for me, after clicking reset, it goes back to the welcome screen.

![image](https://github.com/alephium/extension-wallet/assets/496778/54bf21d2-85a5-454b-a4df-bdba626e51fa)

Reply:
Nope. Just happened once.